stl/inc/algorithm

```diff
@@ -2671,6 +2671,51 @@ namespace ranges {
 } // namespace ranges
 #endif // __cpp_lib_concepts
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    // ALIAS TEMPLATE swap_ranges_result
+    template <class _In1, class _In2>
+    using swap_ranges_result = in_in_result<_In1, _In2>;
+
+    // VARIABLE ranges::swap_ranges
+    class _Swap_ranges_fn : private _Not_quite_object {
+    public:
+        using _Not_quite_object::_Not_quite_object;
+
+        // clang-format off
+        template <input_iterator _It1, sentinel_for<_It1> _Se1, input_iterator _It2, sentinel_for<_It2> _Se2>
+            requires indirectly_swappable<_It1, _It2>
+        constexpr swap_ranges_result<_It1, _It2>
+        operator()(_It1 _First1, _Se1 _Last1, _It2 _First2, _Se2 _Last2) const {

```

Comment:
that is a strange formatting.

Maybe put the operator() on the line above?

Reply:
The formatting trick I use is to delete the `// clang-format off` comment, comment the requires-clause (`// requires ...`), run clang-format on the file, uncomment the requires-clause and restore its extra indent, and restore `clang-format off`. In this case I suspect clang-format will move `operator()(` to the previous line.

Reply:
Too bad that no one else seems to be interested in reviewing the clang-format patches for requires clauses.